website/src/components/pageStateSelectors/WasapPageStateSelector.tsx

```diff
@@ -64,7 +64,44 @@ export function WasapPageStateSelector({
                     </label>
                 </div>
             </Inset>
-            <SelectorHeadline>Mutation selection</SelectorHeadline>
+            <SelectorHeadline
+                info={
+                    <div className='relative p-8'>
+                        <form method='dialog'>
+                            <button className='btn btn-sm btn-circle btn-ghost absolute top-2 right-2'>✕</button>
+                        </form>
+                        <h1 className='mb-2 text-xl font-semibold'>Exploration modes</h1>
```

Comment:
Let's extract this whole info modal content into a separate component? 

Usually, when reading the code, this will be unimportant details ("some info text, let's skip over it..."). IMO it takes too much visual space here.